shell_integration/nautilus/syncstate.py

```diff
@@ -86,9 +87,9 @@ def addListener(self, listener):
     def _connectToSocketServer(self):
         try:
             self._sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
-            postfix = "/" + appname + "/socket"  # Should use os.path.join instead
-            sock_file = get_runtime_dir() + postfix
-            print ("Socket: " + sock_file + " <=> " + postfix)
+            postfix = os.path.join(appname, "socket")
```

Comment:
I looked more closely on this and the check for a valid "Sock-File" isn't making sense. There is a `try` to catch illegal accesses to an invalid socket file (for example permission denied). The code just checked if `get_runtime_dir()` returned empty string and deemed it invalid. All else returns of `get_runtime_dir()` would succeed. I kill the code and make it clean.